modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -23,6 +23,18 @@ You run a DPDK checkup by performing the following steps:
 * You have installed the OpenShift CLI (`oc`).
 * You have configured the compute nodes to run DPDK applications on VMs with zero packet loss.
 
++
+[IMPORTANT]
+====
+The traffic generator pod used by this checkup requires elevated privileges:
+1. It runs as root.
+2. It has a bind mount to the node's file-system.
+
+The traffic generator's image is pulled from community sources.
```

Comment:
```suggestion
The container image of the traffic generator is pulled from the upstream Project Quay Container Registry..
```
Btw, we do call this out in step 5 of the checkup procedure, so I am not sure if we need to repeat that here.

Reply:
Done.

Reply:
> Btw, we do call this out in step 5 of the checkup procedure, so I am not sure if we need to repeat that here.

I wanted to emphasize the combination of a pod with high privileges that is pulled from upstream sources.